### PR TITLE
Native SNS and DynamoDB Support

### PR DESCRIPTION
**Pull request by cnuss**
*Description of changes:*

 - Add native request/response handlers for SNS and DynamoDB events

# Checklist

- [x] Tests have been added and are passing
- [x] Documentation has been updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by cnuss** [comment]
@brettstack Could you have a look at this PR? Let me know if you have any comments/suggestions/issues with this!

Thanks!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vendia/serverless-express",
-  "version": "4.3.11",
+  "version": "4.4.0",
```
Comment (brettstack) [inline package.json]: Please revert this, as it will be taken care of automatically by CI.

**Review by brettstack — COMMENTED** [review]
This looks great! Thanks! Just one thing to remove is the manual version update in package.json. I'll merge this as a `feat:` commit message and it'll automatically get published as that version. Do you have examples you're able to commit also? Thanks again!

**Comment by cnuss** [comment]
@brettstack removed the `package.json` modification of the version number

Re: examples -- I'll whip together an example, but could I do it as another PR, or would you like one as part of this PR?

**Review by brettstack — APPROVED** [review]

**Comment by brettstack** [comment]
Thanks @cnuss! This is a great addition. Best of luck with Scaffoldly.

**Comment by cnuss** [comment]
thanks @brettstack! serverless-express has been amazing

if you'd like to connect and learn a little about Scaffoldly, I'm looking for some early feedback and would be happy to show you around. hit me up! my LinkedIn and Twitter details are in my bio

**Pull request merged**